src/bundle.rs

```diff
@@ -391,6 +419,24 @@ mod tests {
             &serde_json::to_string(&bundle).unwrap(),
             r#"{"txs":["0x01","0x02"],"revertingTxHashes":["0xf2ee15ea639b73fa3db9b34a245bdfa015c260c598b211bf05a1ecc4b3e3b4f2"],"blockNumber":"0x2","minTimestamp":1000,"maxTimestamp":2000,"stateBlockNumber":"0x1","timestamp":1000,"baseFee":333333}"#
         );
+
+        let mut bundle = BundleRequest::new()
```

Comment:
Can you push this to a new test case?

Reply:
moved in https://github.com/onbjerg/ethers-flashbots/pull/46/commits/00dab1721631c37907342cc871a94de6f34c60d9